python/py-pew/Portfile

```diff
@@ -17,7 +19,7 @@ categories          python
 platforms           darwin
 license             MIT
 
-python.versions     35 36 37
+python.versions     26 27 33 34 35 36 37
```

Comment:
Please don't add obsolete python versions. The only one I think is acceptable is py27.

Reply:
Ok. Since we had enough of py27, let's not add it either.

So I removed py26, py27, py33, py34 versions. Moden python rulez!